SMLHelper/Handlers/EquipmentHandler.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using SMLHelper.V2.Handlers;
+    using SMLHelper.V2.Interfaces;
+    using SMLHelper.V2.Patchers.EnumPatching;
+    using SMLHelper.V2.Utility;
+
+    /// <summary>
+    /// A handler class for everything related to creating new Equipments.
+    /// </summary>
+    public class EquipmentHandler : IEquipmentHandler
+    {
+        /// <summary>
+        /// Main entry point for all calls to this handler.
+        /// </summary>
+        public static IEquipmentHandler Main { get; } = new EquipmentHandler();
+
+        private EquipmentHandler()
+        {
+            // Hide constructor
+        }
+
+        /// <summary>
+        /// Adds a new <see cref="Equipment" /> into the game.
+        /// </summary>
+        /// <param name="equipmentName">The name of the Equipment. Should not contain special characters.</param>
+        /// <param name="displayName">The display name of the Equipment. Can be anything.</param>
+        /// <returns>
+        /// The new <see cref="Equipment" /> that is created.
+        /// </returns>
+        public EquipmentType RegisterNewEquipmentType(string equipmentName, string displayName)
+        {
+            EquipmentType equipment = EquipmentTypePatcher.AddEquipmentType(equipmentName);
+            LanguageHandler.SetLanguageLine(equipmentName, displayName);
```

Comment:
Does the game use the display name anywhere? if not, then we dont need a `LanguageHandler.SetLanguageLine()` line.
while you're at it, rename the `RegisterNewEquipmentType` method to `AddEquipmentType` for consistency reasons.

Reply:
No, I don't think it uses the display name. That was a little bit of greedy copy-pasting. I also amended the method name.